<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWindow</class>
 <widget class="QMainWindow" name="GraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Semantics Analyzer</string>
  </property>
  <widget class="QWidget" name="_centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="_scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="_scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>348</width>
         <height>508</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="TextLabel" name="_text">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Text Label</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignJustify|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="_verticalLayout">
      <item>
       <widget class="MplWidget" name="_mplWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="_slider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="_menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="_menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="_actionSave_Visible_Chart_as_Image"/>
   </widget>
   <widget class="QMenu" name="_menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="_menuTarget">
     <property name="title">
      <string>Target</string>
     </property>
     <addaction name="_actionClauses"/>
     <addaction name="_actionSuperClauses"/>
    </widget>
    <addaction name="_actionVisible_points"/>
    <addaction name="separator"/>
    <addaction name="_menuTarget"/>
   </widget>
   <widget class="QMenu" name="_menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="_menuVisibility">
     <property name="title">
      <string>Visibility</string>
     </property>
     <addaction name="_actionSD"/>
     <addaction name="_actionSG"/>
    </widget>
    <addaction name="_menuVisibility"/>
   </widget>
   <widget class="QAction" name="_menuHelp">
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="_menuFile"/>
   <addaction name="_menuEdit"/>
   <addaction name="_menuView"/>
   <addaction name="_menuHelp"/>
  </widget>
  <action name="_actionSave_Visible_Chart_as_Image">
   <property name="text">
    <string>Save Visible Chart as Image</string>
   </property>
  </action>
  <action name="_actionSD">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SD</string>
   </property>
  </action>
  <action name="_actionSG">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SG</string>
   </property>
  </action>
  <action name="_actionVisible_points">
   <property name="text">
    <string>Visible points</string>
   </property>
  </action>
  <actiongroup name="_actiongroupTarget">
   <action name="_actionClauses">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Clauses</string>
    </property>
   </action>
   <action name="_actionSuperClauses">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Super clauses</string>
    </property>
   </action>
  </actiongroup>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header location="global">.mpl_canvas</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TextLabel</class>
   <extends>QLabel</extends>
   <header>.text_label</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
